docs/reference/settings/audit-settings.asciidoc

```diff
@@ -40,29 +48,39 @@ by using the following settings:
 // tag::xpack-sa-lf-events-include-tag[]
 `xpack.security.audit.logfile.events.include`::
 (<<dynamic-cluster-setting,Dynamic>>)
-Specifies which events to include in the auditing output. The default value is:
+Specifies the kind of events to print in the auditing output. The default list value
```

Comment:
> Specifies the kind of events to print in the auditing output. 

An `_all` value is mentioned later. Might be nice to have its meaning/use case here too.

Reply:
I have added the mention for `_all` and referred to the `audit-event-types` anchor for the list of permitted values.